modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -704,6 +704,77 @@ public void testClusterChangeTag() throws Exception {
         assertTrue("Tag has not been updated in 10 seconds", tagUpdated);
     }
 
+    /**
+     * Tests idle_verify working on an active cluster with persistence. Works via control.sh.
+     *
+     * @throws Exception if failed.
+     */
+    @Test
+    public void testIdleVerifyOnActiveClusterWithPersistence() throws Exception {
```

Comment:
@J-Bakuli , have you checked, that we have no positive scenario tests for IdleVerify?

Reply:
@shishkovilja, yes, I have searched for such tests, both positive and negative scenarios for IdleVerify - inactive with persistence, active with persistence

Reply:
And are such tests absent?

Reply:
I will delete the active test